src/angular/directives/dir-date-validation.js

```diff
@@ -0,0 +1,27 @@
+const app = angular.module('materia')
```

Comment:
Since all of the elements using this directive have been replaced with React components, this file is no longer necessary. I'll make a separate commit to remove it.